kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection.vue

```diff
@@ -151,7 +179,11 @@
     },
     methods: {
       closeSidePanel() {
-        this.$router.go(-1);
+        if (this.showSearch == true) {
+          this.$router.go(-1);
+        } else {
+          this.showSearch = true;
```

Comment:
So, this was not explicitly listed as acceptance criteria, but in the figma, this panel actually has a back arrow and a close button. Can you check in with @jtamiace about the expected behavior, and potentially update your PR to include this? I _think_ what should happen is that the back arrow should return the user to their previous location, and the close arrow should close the side panel entirely. (cc @nucleogenesis)